constellations/gemini-user/src/account/account.sass

```diff
@@ -0,0 +1,302 @@
+@import "../utils/_constants.sass"
+
+#view-account
+    width: 80%
+    margin: 0 auto $spacing16x
+
+    #tab-container
+        display: flex
+        flex-direction: row
+        align-items: center
+        border-bottom: 1px solid $colorGrayLight
+
+        & div
+            padding: $spacing2x 0
+            flex-grow: 1
+            border-bottom: 2px solid transparent
+            text-align: center
+            transition: 0.3s
+
+        & div:hover
+            background-color: $colorWhiteLight
+            cursor: pointer
+
+        & div.selected
+            border-bottom: 2px solid $colorTurquoise
+
+    .tab-content
+        > div
+            margin: $spacing6x 0 $spacing10x
+
+
+    @media screen and (min-width: $mediaWidth3)
+        .tab-content
+            ul
+                display: flex
+                flex-direction: row
+                border-bottom: 1px solid $colorWhiteLight
+
+                li
+                    padding: $spacing3x 0
+
+                    .classList-item
+                        margin: $spacing2x 0
+                    .classList-item:first-child
+                        margin-top: 0
+                    .classList-item:last-child
+                        margin-bottom: 0
+
+                .li-med
+                    width: $spacing32x
+                .li-large
+                    flex-grow: 1
+
+            ul.three-columns
+                li
+                    width: calc(100% / 3)
+
+            ul.vertical-centered
+                align-items: center
+
+            #password-change
+                width: 60%
+
+                li
+                    padding: $spacing1x 0
+
+                    input
+                        width: calc(100% - 16px)
+
+        .tab-content#settings-tab
+            ul
+                padding: $spacing2x 0
+
+
+    @media screen and (max-width: $mediaWidth1)
+        h2
+            margin-bottom: $spacing2x
+
+        .tab-content
+            text-align: center
+
+            ul
+                display: flex
+                flex-direction: row
+                flex-wrap: wrap
+                border-bottom: 1px solid $colorWhiteLight
+                text-align: left
+
+                li
+                    padding: $spacing1x 0
+
+        .tab-content#payment-tab
```

Comment:
I would split the CSS to their own individual components.
Let account.sass handle CSS that handles anything all the tabs share. For example, the tab switching highlight behavior, etc. should be handled by account.sass. And let account/payment.sass handle everything SPECIFICALLY inside the payment tab content.

Even if the file is small, you can do something like this:

```
#view-account .tab-content#payment-tab
    ...
```